beginner_source/blitz/autograd_tutorial.py

```diff
@@ -1,198 +1,264 @@
 # -*- coding: utf-8 -*-
 """
-Autograd: Automatic Differentiation
-===================================
+A Gentle Introduction to Autograd
+---------------------------------
 
-Central to all neural networks in PyTorch is the ``autograd`` package.
-Let’s first briefly visit this, and we will then go to training our
-first neural network.
+Autograd is PyTorch’s automatic differentiation engine that powers
+neural network training. In this section, you will get a conceptual
+understanding of how autograd works under the hood.
 
+Skip to advanced readings:
```

Comment:
Why isn't this at the end as a "going further" section?

Reply:
Fixed